src/main/webapp/app/exam/participate/exam-start-information/exam-start-information.component.ts

```diff
@@ -45,45 +45,78 @@ export class ExamStartInformationComponent implements OnInit {
         this.prepareInformationBoxData();
     }
 
-    buildInformationBox(boxTitle: string, boxContent: string | number, boxContentComponent?: string): InformationBox {
+    buildInformationBox(boxTitle: string, boxContent: InformationBoxContent, isContentComponent?: boolean): InformationBox {
```

Comment:
```suggestion
    buildInformationBox(boxTitle: string, boxContent: InformationBoxContent, isContentComponent = false): InformationBox {
```
This way you can omit the nullish operator